modules/ROOT/pages/apicat-create-descriptor-file-cli.adoc

```diff
@@ -67,6 +67,36 @@ The presence and content of the designated descriptor file affects the result of
 
 * Existing file with content: The command fails and shows an error message that says the descriptor file already exists.
 
+=== Link Documentation Pages and Images through a Descriptor File
+
+You can add documentation pages and images to your APIs by adding a markdown file in the descriptor. You can visualize the image in the documentation page created for the Exchange Asset.
+
+Example descriptor file:
+
+----
+#%Catalog Descriptor 1.0
+projects:
+  - main: sources/api.raml
+    documentation:
+      - Summary: Summary.md
+      - Use Cases: Use_Cases.md
+    assetId: version-basic-api
+    apiVersion: v4
+    version: 4.0.0
+----
+With the `summary.md` file containing a link to an image (.png, .jpg) and a documentation markdown file:
+----
+This is a caching service that can be used to retrieve picklist values from publishers who wish to provide a caching service for their picklist fields.
+
+![muley logo](muleyLogo.png)
+
+[Read all the Use Cases](Use_Cases.md)
+----
+
+The presence and content of the designated descriptor file affect the result of the command as follows:
+
+* No documentation entry for the referenced file in the descriptor catalog: the link to move between pages in the Asset's portal doesn't work.
```

Comment:
The phrase "the link to move between pages in the Asset's portal doesn't work" should read:
"the link to move between pages in the asset portal doesn't work".

- The word "Asset" should not be capitalized since it does not seem to be a proper name.
- I guess "asset" needs no possessive case (https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference). Not 100% sure, though.